### PR TITLE
Fix/#19

### LINKED ISSUE #19: bug in the color of the fields
### Explanation
you need the fields to be blank, you want the change to be made and the fields not to be red, so that the user doesn't get the impression that they are in error, this error appears in the 2 pages.

### Screenshots

Original:
image
Our: 
image

Original: 
image
Our: 
image

### Checklist

This issue can be closed when the following tasks are complete:

- [x] on page 1 there is no red color in the fields.
- [x] on page 2 there is no red color in the fields.

### References
Original: https://www.figma.com/proto/9hPU5c4waRbBOd8msfEnKa/Search_by_time_github?node-id=2%3A8&scaling=scale-down-width&page-id=0%3A1

Our: https://azordev.github.io/search-by-time-github/

### PR DESCRIPTION
**Pull request by mdrianislam0or1**
### Description

Remove the red color in the fields from two page.
Add the screenshorts
redremove1
redremove2

fixed #19

### CODE REVIEW

**Review by Israel-Laguan — APPROVED** [review]

**Comment by mdrianislam0or1** [comment]
I solved this issue but it's said conflicts again and again.

**Comment by Israel-Laguan** [comment]
@Katsu08 can you check this one please, so @mdrianislam0or1 can unblock?

**Review by e-azocar — APPROVED** [review]

**Comment by e-azocar** [comment]
It's ok, but it's necessary solve the conflicts

**Comment by heliomar-pena** [comment]
> @Katsu08 can you check this one please, so @mdrianislam0or1 can unblock?

Hello, I think there's an error, since the red color is ok, the problem really is that when the user enter on the page for first time he doesn't should see any error.

I see that the error color is changed from red to black, but I think it isn't fixing the problem

image

Since I'm seeing the error message above of the fields

File: src/js/utils.js [hunk 1]
```diff
@@ -109,4 +109,4 @@ export const getFormValues = (form) => {
   }
 
   return values;
-}
\ No newline at end of file
+};
```
Comment (Israel-Laguan) [inline src/js/utils.js:112]: Please add EOF line

**Comment by Israel-Laguan** [comment]
Please check conflict, @e-azocar when you can try to help with rebase

**Review by Israel-Laguan — APPROVED** [review]

**Pull request merged**